src/rss/filing_tracker.py

```
@@ -0,0 +1,290 @@
+                **self.stats,
+                "unique_cases": len(set(f['case_number'] for f in self.filings.values())),
+                "corporate_filings": sum(1 for f in self.filings.values() if f.get('is_corporate', False)),
+                "individual_filings": sum(1 for f in self.filings.values() if not f.get('is_corporate', False))
+            }
+    
+    def export_to_csv(self, output_file: Path) -> None:
+        """Export filings to CSV format"""
+        import csv
+        
+        with self._lock:
+            with open(output_file, 'w', newline='', encoding='utf-8') as f:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)